internal/commands/sync_with_local_test.go

```diff
@@ -2,6 +2,8 @@ package commands_test
 
 import (
 	"errors"
+	"github.com/pivotal-cf/kiln/pkg/component"
+	fetcherFakes "github.com/pivotal-cf/kiln/pkg/component/fakes"
```

Comment:
Much better naming scheme!  Can we favor this over "fakes2" everywhere we have to disambiguate package names?